fs/zipfs.py

```diff
@@ -290,13 +311,15 @@ class ReadZipFS(FS):
     }
 
     @errors.CreateFailed.catch_all
-    def __init__(self, file, encoding="utf-8"):
-        # type: (Union[BinaryIO, Text], Text) -> None
+    def __init__(self, file, encoding="utf-8", passwd=None):
```

Comment:
Can you rename `passwd` to `password`. I prefer to avoid abbreviations where possible.